src/main/java/org/folio/inventory/support/http/server/JsonResponse.java

```diff
@@ -23,6 +23,10 @@ public static void created(HttpServerResponse response,
     response(response, body, 201);
   }
 
+  public static void createdWithEmptyResponse(HttpServerResponse response) {
+    response(response, new JsonObject(), 201);
+  }
```

Comment:
By empty response, is the intention for a response with a JSON object with no properties or a response with no body at all?

Reply:
Empty response (without empty JSON). Fixed.